java/solrmarc_mixin/src/de/unituebingen/ub/ubtools/solrmarcMixin/TuelibAuthMixin.java

```diff
@@ -111,13 +111,18 @@ public String getYearRange(final Record record) {
                     return "[" + matcher.group(1) + " TO " + matcher.group(2) + "]";
             }
         }
-
         return null;
     }
     
     public String getAuthorityType(final Record record) {
-        if (record.getVariableFields("100").size() > 0)
+        if (record.getVariableFields("100").size() > 0) {
```

Comment:
It might be slightly better to store the result of getVariableFields in a variable and re-using it instead of calling it twice